stablehlo/dialect/StablehloOps.td

```diff
@@ -2788,12 +2719,6 @@ def StableHLO_DynamicGatherOp: StableHLO_Op<"dynamic_gather",
     DefaultValuedAttr<BoolAttr, "false">:$indices_are_sorted
   );
   let results = (outs HLO_Tensor);
-
-  let extraClassDeclaration = [{
-    static bool isCompatibleReturnTypes(TypeRange l, TypeRange r) {
-      return succeeded(mlir::verifyCompatibleShapes(l, r));
-    }
-  }];
 }
 
 def StableHLO_DynamicConvOp : StableHLO_Op<"dynamic_conv", [NoSideEffect]> {
```

Comment:
Also, should we make the same change for CHLO?

Reply:
Sure.